### ISSUE TITLE
Inconsistent directory icons since 18.03. STABLE update

### ISSUE THREAD

**Issue by q-pa** [lumina-desktop/lumina#576]
I noticed that since the update to 18.03 STABLE some directories now show with general file icons instead of directory icons. In the directory tree view all icons are correct, though.

![directory_icons_18 03](https://user-images.githubusercontent.com/3322553/38800783-70abfdc0-4168-11e8-87ba-0ccf3e243013.png)

![directory_icons_tree_18 03](https://user-images.githubusercontent.com/3322553/38800792-76856b78-4168-11e8-96cf-1f8ef4f4bae1.png)

**Comment by q-pa** [comment]
I forgot to mention that I am seeing this issue on both of my computers.

**Comment by beanpole135** [comment]
Looking into this. It might be another side-effect of some changes in how Qt 5.9.4 loads icons from the current theme.

**Comment by beanpole135** [comment]
Ok, I think I fixed the icons which should be used in lumina-fm.
In our file information class, I had to add a list of possible icons for each of the special types of files/dirs, and then use the first one that exists in the current icon theme. The reason for this is that it appears that different themes use slightly different icon names for the same thing, so we need to be able to input all the variations on the icon name to make it support all the major icon themes.

https://github.com/lumina-desktop/lumina/commit/3a53ebfe91ae0aad59c071c48076fa57256920c5
